cerebrium/logging/censius.mdx

```diff
@@ -0,0 +1,74 @@
+---
+title: "Censius"
+description: "Censius is a monitoring tool that allows you to track your model performance in production environments."
+---
+
```

Comment:
maybe give a bit of a one line intro. "To get started with setting up Censius start with the instructions below"